Library/Homebrew/utils/github.rb

```diff
@@ -506,14 +506,16 @@ def self.get_repo_license(user, repo)
     nil
   end
 
+  def self.pull_request_title_regex(name, version = nil)
+    return /(^|\s)#{Regexp.quote(name)}(:|,|\s|$)/i.freeze if version.blank?
```

Comment:
Is `Regexp.quote` the same as `Regexp.escape`?

Reply:
I think so. As mentioned above, I lifted these regexes from the ones we were already using.

Reply:
Ok, no worries, a comment explaining it a bit might be helpful (assuming you understand it 😅)